.pre-commit-config.yaml

```diff
@@ -42,15 +42,16 @@ repos:
             - id: codespell
               args: ["--config pyproject.toml"]
               additional_dependencies: ["tomli"]
+      - repo: https://github.com/rapidsai/pre-commit-hooks
+        rev: v0.2.0
+        hooks:
+            - id: verify-copyright
+            - id: verify-alpha-spec
       - repo: https://github.com/rapidsai/dependency-file-generator
         rev: v1.13.11
         hooks:
             - id: rapids-dependency-file-generator
               args: ["--clean"]
-      - repo: https://github.com/rapidsai/pre-commit-hooks
-        rev: v0.0.3
-        hooks:
-            - id: verify-copyright
```

Comment:
The updates to RAPIDS `pre-commit-hooks` meant that `branch-24.10` is out-of-date relative to the `pre-commit-hooks` expectations. So resolving conflicts in this case also means using the new `pre-commit-hooks`

Maybe it is worth figuring out a better way to update pre-commit hooks so that we are not accidentally introducing conflicts in bot forward (or backward) auto-mergers